bin/deploy

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -eu
+
+while :; do
+  while getopts ":a:e:p:r:t:" arg; do
+    case $arg in
+      a)
+        auth_file_key="$OPTARG"
+        ;;
+      e) 
+        enc_auth_file="$OPTARG"
+        ;;
+      p)
+        source_path="$OPTARG"
+        ;;
+      r)
+        run_tests="$OPTARG"
+        ;;
+      t)
+        test_level="$OPTARG"
+        ;;
+      *)
+        echo Ignoring unknown option: $OPTARG
+        ;;
+    esac
+  done
+  OPTIND=$((OPTIND+1))
+  [ $OPTIND -gt $# ] && break
+done
+
+if [ "${test_level}" = "RunSpecifiedTests" ] && [ "${run_tests}" = "NotSet" ]
+then
+	echo "ERROR: Please specify Apex unit tests with run_tests argument"
+	exit 1
+fi
+
+# Decrypt Auth URL File
+openssl enc -d -aes-256-cbc -md md5 -in ${enc_auth_file} -out auth_url.txt -k ${auth_file_key}
+
+# Authenticate Target Org
+sfdx force:auth:sfdxurl:store -f auth_url.txt -d -a TargetOrg && rm auth_url.txt
+
+# Run Deployment
+if [ "${test_level}" = "RunSpecifiedTests" ]
```

Comment:
If we are working on a focused aspect of a very large org it could be easier and quicker to specify the Apex tests to run as opposed to all local tests when doing deployments/validations

Reply:
I've def experienced that time diff w/ the Alliance project.